src/PackageVersions/VersionsCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace PackageVersions;
+
+use Composer\Command\BaseCommand;
+use Symfony\Component\Console\Input\InputArgument;
```

Comment:
Well, I think this extension is worthy enough to be added to Composer itself, so maybe it would be OK to keep this waiting until that moment :)